### PR TITLE
Use `rust:alpine` container for small TCB builds

### PR DESCRIPTION
**Pull request by billionairiam**

### CODE REVIEW

File: .github/workflows/alpine.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Prepare alpine dockerfile
+      run: echo "From rust:1.72.0-alpine3.18" >> Dockerfile; echo "RUN apk add musl-dev openssl-dev openssl-libs-static protobuf-dev git && git clone https://github.com/intel/ACON.git" >> Dockerfile; echo "RUN cd ACON/acond && CARGO_BUILD_RUSTFLAGS=-Crelocation-model=static cargo build --release --target x86_64-unknown-linux-musl && CARGO_BUILD_RUSTFLAGS=-Ctarget-feature=-crt-static cargo build --release" >> Dockerfile
```
Comment (binxing) [inline .github/workflows/alpine.yml:20]: Multiple `echo`s can be combined into one `echo -e`.

And shall we use the latest rustc and Alpine?

And the 2 builds will collide on the binary name. Doesn't it matter?

File: .github/workflows/alpine.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - name: Build acond in alpine docker
```
Comment (binxing) [inline .github/workflows/alpine.yml:21]: The other job is called `Debian build`, let call this `Alpine build` to be consistent.

File: .github/workflows/alpine.yml (hunk 1 above)
Comment (billionairiam) [reply .github/workflows/alpine.yml]: Fixed the echo with `echo -e`
Fixed the docker with `From rust:alpine`
Fixed the build process with `cargo clean`

File: .github/workflows/alpine.yml (hunk 2 above)
Comment (billionairiam) [reply .github/workflows/alpine.yml]: Fixed!

**Pull request merged**